src/users.py

```diff
@@ -190,13 +193,17 @@ def _reset(evt, var):
             _users.discard(user)
     _ghosts.clear()
 
+def _swap_player(evt, var, old_user, user):
+    """Marks the user as no longer being a ghost, if they are one."""
```

Comment:
Docstrings should be imperative, i.e. "Mark the user" and not "Marks the user". Yes, other places are wrong, but that's just another reason to fix those places \o/